feat(exp): document maximum duration for application bootstrap (#175)
anbox-cloud/discourse-docs: exp/applications.md

[note type="information" status="Note"]An application crash or ANR upon APK installation will cause the bootstrap process to terminate abnormally and the status of application is set to `error` too.[/note]

> Furthermore the bootstrap process is limited to a maximum duration of 15 min. Afterwards the bootstrap process will be aborted and the container will be marked with status `error`.

Reviewer: ```suggestion
The bootstrap process is limited to a maximum duration of 15 minutes. If it takes longer, the bootstrap process is aborted and the container is marked with the status `error`.
```
